plugins/katello/3.3/release_notes/release_notes.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: plugins/katello/documentation
+title: Release Notes
+version: 3.3
+---
+
+# Katello Nightly () Release Notes
+
+{% if page.version and page.version != '3.3' %}
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
+{% else %}
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+{% endif %}
+
+## Features
+
+## Deprecation Warnings
+
+## Contributors
+
```

Comment:
I think that might be better as a separate commit (easier to review and comment on).